packages/salesforce/src/Adaptor.js

```diff
@@ -709,11 +711,11 @@ function getConnection(state, options) {
   const apiVersionRegex = /^\d{2}\.\d$/;
 
   if (apiVersion && apiVersionRegex.test(apiVersion)) {
-    console.log('Using Salesforce API version', apiVersion);
+    console.log('Using Salesforce API version:', apiVersion);
     options.version = apiVersion;
   } else {
-    console.log('apiVersion is not defined');
-    console.log('We recommend using Salesforce API version 52.0 or latest');
+    options.version = '47.0';
+    console.log('Using Salesforce API version:', options.version);
```

Comment:
A small nitpick but since the messages are the same, can we restructure like this?:
```
if (apiVersion && apiVersionRegex.test(apiVersion)) {
   options.version = apiVersion;
} else {
   options.version = '47.0';
}
console.log('Using Salesforce API version:', options.version);
```

Reply:
Done ✅